lib/Runtime/Library/ArrayBuffer.h

```diff
@@ -166,7 +166,7 @@ namespace Js
         static int GetBufferOffset() { return offsetof(ArrayBuffer, buffer); }
 
         virtual void AddParent(ArrayBufferParent* parent) override;
-#if _WIN64
```

Comment:
Comment below mentions amd64 -- should this be _M_X64 instead? (theres a similar use in SharedArrayBuffer.h)

Reply:
The numbers there are not vendor dependent.